### ISSUE TITLE
Relative Textgrößen machen Textfehler in MS Office

### ISSUE THREAD

**Issue by squeakyflamingo** [ReneDens/Bibliothek-taktische-Zeichen#24]
Dadurch, dass die Texte (bspw. in Person-EL) mit relativen Größen angelegt sind, werden sie in MS Office nicht schön dargestellt.

In Word:
![In Word](https://user-images.githubusercontent.com/34474194/149017080-1c5768d1-cf14-4118-b157-075e16cc7bf5.png)

In Firefox:
![In Firefox](https://user-images.githubusercontent.com/34474194/149017183-562ba428-80c2-49d1-86f3-59508940ed28.png)

Code "relative Textgröße":
```svg
<text xmlns="http://www.w3.org/2000/svg" x="128" y="128" font-family="Roboto Slab Bold" font-size="300%" dominant-baseline="central" fill="" text-anchor="middle" id="text15" style="-inkscape-font-specification:'Roboto Slab Bold';font-family:'Roboto Slab';font-weight:bold;font-style:normal;font-stretch:normal;font-variant:normal;">EL</text>
```

**Comment by jonas-koeritz** [comment]
Das ist in meinem Repo (dass die Basis für diese Zeichen darstellt) größtenteils gefixt.

**Comment by squeakyflamingo** [comment]
Aber da fehlen mir die ganzen kleinteiligen Zeichen der Wasserrettung. :|

**Comment by jonas-koeritz** [comment]
@ReneDens gibt's da eventuell nen pull request?

**Comment by ReneDens** [comment]
@jonas-koeritz Da muss ich mal kucken wie ich das noch gemerget bekomme. Danke für den Tipp.

**Comment by squeakyflamingo** [comment]
Vllt. bastle ich mir auch mal aus euren Bibliotheken eine Mischung. :D
Aber falls es schon gefixt existiert, freue ich mich da auch.

**Comment by jonas-koeritz** [comment]
Eventuell kopiere ich auch von hier einfach die fehlenden und markiere dich als Autor, natürlich nur wenn das für dich klar geht.

**Comment by squeakyflamingo** [comment]
Falls das nicht zu viel Aufwand ist, fände ich es cool, wenn das WR-Symbol von hier gleich übernommen würde. @ReneDens hat es, glaube ich, näher an die DV102-Beispiele geschafft.

**Comment by ReneDens** [comment]
@jonas-koeritz Klar, gerne, bedien dich und merge/kopiere durch die Gegend. Dafür isses ja freie Lizenz.
N paar sind vielleicht bayerische Spezialfälle, aber der Rest ist ja universell.
Irgendwann wenn ich mal mehr Zeit hab melde ich mich mal bei dir, damit wir vielleicht unsere Kräfte komplett in einem Projekt bündeln können.

@ squeakyflamingo Mein Wasserrettungssymbol ist dafür auch komplexer gebaut und könnte evlt non Standard SVG Formen enthalten, weil es nicht zu Fuss, sondern mit Inkscape gebaut ist.

**Comment by squeakyflamingo** [comment]
Das WR-Symbol kann man relativ easy etwas aufräumen. Und dann sieht es eigentlich ziemlich standardkonform aus. :)
Keine Ahnung, wie easy das in den j2-Build implementierbar ist, aber vllt. ist es ja tatsächlich mit Copy-Paste getan.

```svg
<rect x="176.5" y="-4" width="34" height="34" transform="rotate(45)" fill="none" stroke="#000000" stroke-width="5"/>

<path d="m 99.045829,98.736424 c 0,0 -0.264149,-13.32437 9.460921,-13.97747 9.77075,-0.656146 10.3562,7.265035 10.3562,7.265035 0,0 0.30639,6.735827 8.11679,6.735827 7.81038,0 8.37446,-6.67634 8.37446,-6.67634 0,0 1.94572,-7.579966 10.92923,-7.312057 9.87626,0.294533 9.94718,14.049006 9.94718,14.049006"
fill="none" stroke="#000000" stroke-width="5" />

<path d="m 99.770777,115.07915 c 0,0 -0.264149,-13.32436 9.460923,-13.97747 9.77075,-0.65614 10.3562,7.26504 10.3562,7.26504 0,0 0.30639,6.73583 8.11678,6.73583 7.81039,0 8.37447,-6.67634 8.37447,-6.67634 0,0 1.94572,-7.57997 10.92922,-7.31206 9.87626,0.29453 9.94718,14.04901 9.94718,14.04901"
fill="none" stroke="#000000" stroke-width="5" />
```

Als PoC mit ein paar Minuten händischem SVG-Code-Geschubse.
![PoC](https://user-images.githubusercontent.com/34474194/149023765-992a989f-305d-441f-b6d9-ab19a094326d.png)
